### DISCUSSION TITLE
please refund me

### DISCUSSION THREAD

**Discussion by mamglezcom** [aclap-dev/video-downloadhelper#1287] [License & Purchase requests]
### What's your issue?

hi, I have buyed on 20/09/2024 a firefox premium license, but it only save a partial video of about 10 minutes. please refund money, my email is contacto@mamglez.com
in addition, confirmation email never arrived.

### The 10 first digits (only 10) of your license number or email address

2c176e7ced

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
